integration_test/third_party_apps_data/simulacra.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"context"
+	"embed"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/ops-agent/apps"
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator"
+	"github.com/GoogleCloudPlatform/ops-agent/integration_test/agents"
+	"github.com/GoogleCloudPlatform/ops-agent/integration_test/logging"
+	"github.com/google/uuid"
+
+	"github.com/GoogleCloudPlatform/ops-agent/integration_test/gce"
+)
+
+//go:embed applications
```

Comment:
Yep, I agree. I ended up going with what you suggested. Instead of using embed.FS, we take in a path to the third party apps directory. We have it set with a default value. The default value assumes the working directory is the root of the repo.